recipes-kernel/linux-yocto/linux-yocto/lockdep.cfg

```diff
@@ -0,0 +1,99 @@
+CONFIG_EXPERT=y
+CONFIG_KALLSYMS_ALL=y
+# CONFIG_DEBUG_PERF_USE_VMALLOC is not set
```

Comment:
If these options are not intentionally to be specifically turned off, you can remove this to leave them as-is.